examples/asr/jasper_an4.py

```diff
@@ -17,64 +17,68 @@
     process_evaluation_epoch,
     word_error_rate,
 )
+from nemo.core import NeuralGraph
 from nemo.utils import logging
 from nemo.utils.lr_policies import CosineAnnealing
 
 
 def create_dags(model_config_file, vocab, args, nf):
 
-    # Create a data_layer for training.
-    data_layer = nemo_asr.AudioToTextDataLayer.import_from_config(
-        model_config_file,
-        "AudioToTextDataLayer_train",
-        overwrite_params={"manifest_filepath": args.train_dataset, "batch_size": args.batch_size},
-    )
+    with NeuralGraph() as g0:
```

Comment:
Tip: you do not need to "instantiate" modules within the graph scope, generally it has no effect